ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -655,7 +654,7 @@ def main(argv):
 
     analyzer = AnsibleLogAnalyzer(run_id, verbose, start_marker)
 
-    log_file_list = filter(None, log_files_in.split(tokenizer))
```

Comment:
>filter [](http://example.com/codeflow?start=20&length=6)

Is it easier to `list(filter())` ?

Reply:
Yeah, `list(filter())` is easier to read, changed to that syntax.